fe/fe-core/src/main/java/org/apache/doris/analysis/ShowViewStmt.java

```diff
@@ -114,20 +114,20 @@ public void analyze(Analyzer analyzer) throws AnalysisException, UserException {
             View view = (View) table;
             List<TableRef> tblRefs = Lists.newArrayList();
             // get table refs instead of get tables because it don't need to check table's validity
-            getTableRefs(view, tblRefs);
+            getTableRefs(analyzer, view, tblRefs);
             for (TableRef tblRef : tblRefs) {
-                tblRef.analyze(analyzer);
+                tblRef.getName().analyze(analyzer);
```

Comment:
Why not just call `tblRef.analyze(analyzer);`?

Reply:
Because we only need TableName to compare. Besides, there are many unnecessary processes in analyze() in TableRef.